src/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -15,8 +15,17 @@ namespace System.IO
 {
     public class FileSystemAclExtensionsTests
     {
+        #region Private members
```

Comment:
Why do we need all these regions?

Reply:
For collapsing sections I am not working on and organizing the APIs better.